README.md

```diff
@@ -75,10 +75,10 @@ is also available and can be run on any system with a supported Java runtime ins
 
 **Note for macOS users**:
 The Logisim-evolution.app is not code-signed.
-The first time you launch it, you should do so by ctrl- or right-clicking the app and selecting `Open`,
- which will give you a panel that allows you to open it.
-Depending on your security settings, you may also get a panel asking if you wish to allow it to accept network connections.
-You can click `Deny`.
+The first time you launch it, you should do so by holding <kbd>CTRL</kbd> while clicking or clicking by right mouse button
+the app and selecting `Open`, which will give you a panel that allows you to open it. Depending on your security settings,
```

Comment:
I propose for the sentence:

"When launching the application for the first time, you will have to start it via the `Open` entry in the application icon's context menu in the macOS Finder. This is either done by clicking the application icon with the right mouse button or holding down <kdb>control ^</kbd> while clicking the icon with the left mouse button. This will open a panel asking you whether you are sure to launch the application."

@davidhutchens: Is this clearer. Do you have any suggestions to simplify it?

Reply:
I think @maehne's wording is clearer. But I would change "asking you whether you are sure to launch the application" to "asking you to verify that you wish to launch the application".

I'm not sure about using the ^ with control. I would either just say "control" or use <kbd>CTRL</kbd>. I note that my Mac keyboard actually labels the key "control", so maybe <kbd>control</kbd>?

Reply:
Thanks @davidhutchens for your feedback! I like your suggestion.